lib/puma/server.rb

```diff
@@ -424,7 +425,7 @@ def process_client(client, buffer)
 
             ThreadPool.clean_thread_locals if clean_thread_locals
 
-            unless client.reset(@status == :run)
+            unless client.reset(@status == :run && @fast_track_ka_timeout)
```

Comment:
I would say this: https://github.com/puma/puma/pull/1225/files/b373d26a37eaac35ed943c72b183b11f99406104#diff-ef4c610f0ce605eb90f2810856394fd5R487 is an accessor. I'm asking what the purpose of the instance variable is when you could just call `fast_track_ka_timeout`.